File: app/src/main/java/my/edu/tarc/jobseek/ui/headHunting/Candidate.java
package my.edu.tarc.jobseek.ui.headHunting;

import java.lang.reflect.Array;

public class Candidate {
    String name;
    String position;
    String location;
    Array skills;

    public Candidate(String name, String position,
                     String location, Array skills) {
        this.name = name;
        this.position = position;
        this.location = location;
        this.skills = skills;
    }

    public String getName() {
        return name;
    }

    public String getPosition() {
        return position;
    }

    public String getLocation() {
        return location;
    }

    public Array getSkills() {
        return skills;
    }
}

File: app/src/main/java/my/edu/tarc/jobseek/headHunting/Candidate.kt
package my.edu.tarc.jobseek.headHunting

import android.os.Parcel
import android.os.Parcelable
import android.provider.ContactsContract.CommonDataKinds.Email
import androidx.room.Entity
import androidx.room.PrimaryKey


data class Candidate(
    var name: String ?= null,
    var school: String ?= null,
    var levelSchool: String ?= null,
    var cGPA: Float ?= null,
    var field: String ?= null,
    var experience: String ?= null,
    var location: String ?= null,
    var phoneNum: String ?= null,
    var email: String ?= null,
    )